ansible/roles/cartodb.app/templates/app_config.yml.j2

```diff
@@ -11,11 +11,11 @@ defaults: &defaults
   session_domain:     '.localhost.lan'
   # If activated, urls will use usernames in format //SESSION_DOMAIN/user/USERNAME and ignore subdomains if present
   subdomainless_urls: false
-  http_port:           3000 # nil|integer. HTTP port to use when building urls. Leave empty to use default (80)
+  http_port:           {{ cartodb_http_port }} # nil|integer. HTTP port to use when building urls. Leave empty to use default (80)
```

Comment:
Yeah, being explicit with the port 80 so that this var can be used in multiple places without extra logic. The `:80` is stripped from the url, the application works as expected.